docs/src/reference/lib/changelog.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+.. _lib-21.9.0:
+
+21.9.0 (UNRELEASED)
+-------------------
+
+.. start-21.9.0
+
+Changed
+*******
+
+* Klio's config file is now read from disk once per worker instead of once per thread (See `PR 218 <https://github.com/spotify/klio/pull/218>` _).
```

Comment:
it looks like there's a space between the 2nd back tick and the underscore for the link

Reply:
Ah yeah the link is broken: https://klio--218.org.readthedocs.build/en/218/reference/lib/changelog.html#changed

Reply:
Fixed!